lib/stimulus_reflex/sanity_checker.rb

```diff
@@ -62,6 +64,31 @@ def check_javascript_package_version
     end
   end
 
+  def check_new_version_available
+    return unless Rails.env.development?
+    return if StimulusReflex.config.on_new_version_available == :ignore
+    if not_prerelease
+      begin
+        latest_version = open("https://raw.githubusercontent.com/hopsoft/stimulus_reflex/master/LATEST", open_timeout: 1, read_timeout: 1).read.strip
+        if latest_version != StimulusReflex::VERSION
+          puts <<~WARN
+
+            There is a new version of StimulusReflex available!
+            Current: #{StimulusReflex::VERSION} Latest: #{latest_version}
+            It is very important that you update BOTH Gemfile and package.json
+            Run `bundle install && yarn install` to complete the upgrade.
+
+          WARN
+          exit if StimulusReflex.config.on_new_version_available == :exit
+        end
+      rescue
+        puts "StimulusReflex #{StimulusReflex::VERSION} update check skipped: connection timeout"
+      end
+    else
+      puts "StimulusReflex #{StimulusReflex::VERSION} update check skipped: pre-release build"
```

Comment:
Found the branching here a bit hard to read, maybe do line #88 as a guard clause instead.

Reply:
Ah, no, I realize with the puts it a bit hard to do a guard clause.